src/Cli/dotnet/Commands/Pack/PackCommandParser.cs

```diff
@@ -60,6 +64,20 @@ internal static class PackCommandParser
 
     public static readonly Option<string[]> TargetOption = CommonOptions.RequiredMSBuildTargetOption("Pack", [("_IsPacking", "true")]);
 
+    public static readonly Option<string> VersionOption = new ForwardedOption<string>("--version")
+    {
+        Description = CliCommandStrings.PackCmdVersionDescription,
+        HelpName = CliCommandStrings.PackCmdVersion,
+        Hidden = false
+    }.ForwardAsSingle(o => $"-Version={o}");
```

Comment:
I'm very surprised that we don't have a common base for `--version` already :D One more for the pile then I guess :)

Reply:
Since you're not forwarding to anywhere else, this can just be a 'normal' Option. I'd also strongly suggest parsing into an actual version structure at the CLI level (e.g. making this an `Option<NuGetVersion>` or similar) so that we have consistent option-parsing-failure handling across the CLI.  In general, we really do not want to keep making `string` options at the CLI layer.